src/pages/components/icon-button.mdx

```diff
@@ -1,15 +1,117 @@
 ---
 title: Icon Button
 description: >
-  Description will be here
+  Icon Buttons (like Buttons) let users take action. They are used for repeated or persistent
+  actions on a page and lack labels to simplify the UI.
```

Comment:
This isn't entirely accurate, as they should have ARIA labeling and hover tooltips in place. Perhaps

"lack _visible_ label to simplify the UI."

(emphasis mine, don't include in the rendered site)